pkg/minikube/cluster/cluster_windows_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"runtime"
+	"path/filepath"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"github.com/docker/machine/drivers/hyperv"
+)
+
+var machineConfig = MachineConfig{
+	MinikubeISO: "https://github.com/fake/boot2docker.iso",
+	Memory: 2048,
+	CPUs: 2,
+	DiskSize: 10000,
+	VMDriver: "hyperv",
+}
+
+func TestCreateHypervHost(t *testing.T) {
+	if runtime.GOOS != "windows" {
```

Comment:
@hferentschik Didn't know that :smile:  Thanks for sharing the link.

Updated the code now.